cmd/thanos/main.go

```diff
@@ -74,7 +75,7 @@ func main() {
 	cmds := map[string]setupFunc{}
 	registerSidecar(cmds, app)
 	registerStore(cmds, app, "store")
-	registerQuery(cmds, app, "query")
+	registerQuery(cmds, app)
```

Comment:
I think this makes sense but inconsistent to other commands (:

Reply:
Isn't it just because we are in the middle of moving to the readiness probers in each component? Then they will all look the same :stuck_out_tongue:

Reply:
:D Exactly, in the end, they'll all look same.

Reply:
We are! let's go with this :+1: